### PR TITLE
feature/1100 Settlement By Currency Wrap-Up

### PR DESCRIPTION
**Pull request by ggrg**
- Bring documentation up-to-date with settlement-v2 implementation

### CODE REVIEW

File: mojaloop-technical-overview/central-settlements/settlement-process/assets/diagrams/sequence/seq-settlement-6.2.1.plantuml [hunk 1]
```diff
@@ -137,7 +137,7 @@ activate OPERATOR
             {
                 errorInformation: {
                     "errorCode": <integer>,
-                    "errorDescription": "Invalid window(s) found"
+                    "errorDescription": "Inapplicable windows found: K, L, M"
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-settlements/settlement-process/assets/diagrams/sequence/seq-settlement-6.2.1.plantuml]: What do you mean by "K, L, M"?

Comment (ggrg) [reply mojaloop-technical-overview/central-settlements/settlement-process/assets/diagrams/sequence/seq-settlement-6.2.1.plantuml]: These are the id numbers of the inapplicable windows.

Comment (mdebarros) [inline mojaloop-technical-overview/central-settlements/settlement-process/assets/diagrams/sequence/seq-settlement-6.2.1.plantuml]: Perhaps use something like `"Inapplicable windows found: ${WindowId}, ${WindowId}, ${WindowId}"`?

Comment (ggrg) [reply mojaloop-technical-overview/central-settlements/settlement-process/assets/diagrams/sequence/seq-settlement-6.2.1.plantuml]: Corrected

**Pull request merged**